src/msw/printdlg.cpp

```diff
@@ -964,6 +984,14 @@ bool wxWindowsPrintDialog::ConvertFromNative( wxPrintDialogData &data )
     {
         data.SetFromPage(pd->lpPageRanges[0].nFromPage);
         data.SetToPage(pd->lpPageRanges[0].nToPage);
+
+        wxVector<wxPrintPageRange>& ranges = data.GetPageRanges();
+        ranges.clear();
+        for (DWORD i = 0; i < pd->nPageRanges; i++) {
+            int fromPage = pd->lpPageRanges[i].nFromPage;
+            int toPage = pd->lpPageRanges[i].nToPage;
+            ranges.push_back(wxPrintPageRange(fromPage, toPage));
```

Comment:
If we remove ctor as I suggested, we could just write
```suggestion
            ranges.emplace_back(fromPage, toPage);
```
here.